dev-libs/bglibs/files/bglibs-2.04-gcc14.patch

```diff
@@ -0,0 +1,41 @@
+--- a/include/resolve.h
```

Comment:
Always include references at the top of the patch to any Gentoo bugs or upstream references and so on.

Reply:
None of the 2 bugs linked here are specifically about the issue fixed in this patch. The patch fixes the later issue which is uncovered after fixing the `select()` bug, which I did via `sed` in the ebuild, and that sed line has a link.

I don't know what link to add here for the patch

Reply:
Ah, fair enough. I suppose it's OK to leave it.